docs/references/backend/client/verify-client.mdx

```diff
@@ -17,7 +17,7 @@ function verifyClient: (token: string) => Promise<Client>;
 
 | Name | Type | Description |
 | - | - | - |
-| `token` | `string` | The session token to verify. |
+| `token` | `string` | A JWT token that represents the active client. |
```

Comment:
```suggestion
| `token` | `string` | A JWT that represents the active client. |
```

JWT stands for "json web token"

Reply:
I copied this description straight from the BAPI docs - so I'll update there too
https://github.com/clerk/clerk_go/pull/7323